src/components/liskAmount/index.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import LiskAmount from './';
+
+describe('<LiskAmount />', () => {
+  const normalizeNumber = 100000000;
+  it('should render "12932689.645" as "12,932,689.645"', () => {
```

Comment:
Please use a more descriptive title. I'd suggest:
should normalize "12932689.645" as "12,932,689.645"